lib/shared/bucketing-assembly-script/protobuf/variableForUserParams.proto

```diff
@@ -39,6 +39,10 @@ message NullableCustomData {
   bool isNull = 2;
 }
 
+message ClientCustomData_PB {
```

Comment:
where is this used?

Reply:
for passing data into the setClientCustomData method (which now uses protobuf instead of JSON)